tdrs-backend/docker-compose.yml

```diff
@@ -3,7 +3,7 @@ version: "3"
 
 services:
   zaproxy:
-    image: owasp/zap2docker-stable
+    image: owasp/zap2docker-stable:latest
```

Comment:
updated zap version to latest available. this was primarily to check for any fixes in how false positives are handled.